CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 
 ### Features
 - Add grants to materializations ([dbt-labs/dbt-spark#366](https://github.com/dbt-labs/dbt-spark/issues/366), [dbt-labs/dbt-spark#381](https://github.com/dbt-labs/dbt-spark/pull/381))
+- Add `connection_parameters` for databricks-sql-connector connection parameters ([#135](https://github.com/databricks/dbt-databricks/pull/135))
```

Comment:
@ueshin maybe link to docs for users who don't know what this?

Reply:
Also maybe explain why this is useful?

Reply:
👍